gilbreth_perception/src/voxelizer_node.cpp

```diff
@@ -139,15 +139,15 @@ void cloudCb(const sensor_msgs::PointCloud2ConstPtr &cloud_msg) {
   dat.layout.dim[0].label = "x";
   dat.layout.dim[1].label = "y";
   dat.layout.dim[2].label = "z";
-  dat.layout.dim[0].size = voxel_grid_size;
-  dat.layout.dim[1].size = voxel_grid_size;
-  dat.layout.dim[2].size = voxel_grid_size;
-  dat.layout.dim[0].stride = voxel_grid_size * voxel_grid_size * voxel_grid_size;
-  dat.layout.dim[1].stride = voxel_grid_size * voxel_grid_size;
-  dat.layout.dim[1].stride = voxel_grid_size;
+  dat.layout.dim[0].size = VOXEL_GRID_SIZE;
+  dat.layout.dim[1].size = VOXEL_GRID_SIZE;
+  dat.layout.dim[2].size = VOXEL_GRID_SIZE;
+  dat.layout.dim[0].stride = VOXEL_GRID_SIZE * VOXEL_GRID_SIZE * VOXEL_GRID_SIZE;
+  dat.layout.dim[1].stride = VOXEL_GRID_SIZE * VOXEL_GRID_SIZE;
+  dat.layout.dim[1].stride = VOXEL_GRID_SIZE;
   dat.layout.data_offset = 0;
-  std::vector<int> vec(voxel_grid_size * voxel_grid_size * voxel_grid_size, 0);
-  for (int i = 0; i < voxel_grid_size * voxel_grid_size * voxel_grid_size; i++)
+  std::vector<int> vec(VOXEL_GRID_SIZE * VOXEL_GRID_SIZE * VOXEL_GRID_SIZE, 0);
+  for (int i = 0; i < VOXEL_GRID_SIZE * VOXEL_GRID_SIZE * VOXEL_GRID_SIZE; i++)
```

Comment:
This expression is evaluated on every iteration therefore this triple product is computed every time.  I suggest computing that result upfront using std::pow and storing it onto a variable.